test/qa-tests/jstests/replsets/rslib.js

```diff
@@ -0,0 +1,776 @@
+/* eslint no-unused-vars: 0 no-catch-shadow: 0*/
```

Comment:
The linter complains about some of the variables declared below since they are not assigned values at compile time, so I've disabled the relevant linter checks. This file should be deleted in [TOOLS-2554](https://jira.mongodb.org/browse/TOOLS-2554), anyway!